real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -63,7 +63,7 @@
 
 	public static boolean GUI_ON_DEFAULT = true;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1267658085/issues/source?bid=2938665&fileBranchId=2939970#l64)